mantle/go.mod

```diff
@@ -12,11 +12,11 @@ require (
 	github.com/aliyun/alibaba-cloud-sdk-go v0.0.0-20190929091402-5711055976b5
 	github.com/aliyun/aliyun-oss-go-sdk v2.0.3+incompatible
 	github.com/aws/aws-sdk-go v1.34.28
-	github.com/coreos/container-linux-config-transpiler v0.8.0
+	github.com/coreos/container-linux-config-transpiler v0.8.0 // indirect
```

Comment:
Hmm, do we know why these are still here at all?

Reply:
Could not figure that one out. Could not find any use anywhere anymore :/